tasks/install_shell.sh

```diff
@@ -407,8 +407,9 @@ install_file() {
       ;;
     "deb")
       info "installing puppetlabs apt repo with dpkg..."
-      dpkg -i "$2"
+      dpkg -i --force-conflicts "$2"
       apt-get update -y
+      apt-get install --fix-broken -y
```

Comment:
I'm not sure whether we want to affect *other* packages that might be installed, with a broad `--fix-broken`...

Reply:
Totally not the way to do it... I updated with some logic to handle this differently.